torch/csrc/jit/mobile/flatbuffer_loader.cpp

```diff
@@ -591,7 +592,66 @@ mobile::Module load_mobile_module_from_file(
   fread(data.get(), size, 1, f);
   fclose(f);
 #endif
-  return parse_and_initialize_mobile_module(std::move(data), size, device);
+  return data;
+}
+
+mobile::Module load_mobile_module_from_file(
+    const std::string& filename,
+    c10::optional<c10::Device> device) {
+  auto data = read_file(filename);
+  return parse_and_initialize_mobile_module(
+      std::move(data), /*size=*/0, device);
+}
+
+std::shared_ptr<char> read_stream(std::istream& in) {
+  // get size of the stream and reset to orig
+  std::streampos orig_pos = in.tellg();
+  in.seekg(orig_pos, std::ios::end);
+  const long size = in.tellg();
+  in.seekg(orig_pos, in.beg);
+
+  // read stream
+  std::shared_ptr<char> data(static_cast<char*>(malloc(size)), free); // NOLINT
```

Comment:
use aligned_alloc on x86 it's fine but I think on 32 bit arm it ll give some errors if not aligned to 16.